### ISSUE TITLE
Organize Session Form Into wizard

### ISSUE THREAD

**Issue by mdwiltfong** [mdwiltfong/secret-santa#44]
The form needs to be able to send data only for creating sessions, AND THEN accept invitees if the session was created.

**Comment by ivavay** [comment]
hi

**Comment by mdwiltfong** [comment]
@ivavay if you're thinking about taking on validation, let me know. I have a WIP on it so far [here](https://github.com/mdwiltfong/secret-santa/pull/47)

**Comment by ivavay** [comment]
Okay, let me work on the wizard first!

**Comment by mdwiltfong** [comment]
Another thing I wanted to bring up, is that i'm looking into how the front-end would display validation feedback. Right now i'm trying to think of bring this into the code: https://getbootstrap.com/docs/4.0/components/forms/#validation

**Comment by ivavay** [comment]
Hi! Sorry I didn't get to work on the project this week! I'll carve out some time in the coming weekend to work on it.

**Comment by ivavay** [comment]
Hi! I'm in the process of creating the wizard and just wanted to make sure that it is a two-step form. So the first step would have all the form inputs listed [here](https://github.com/mdwiltfong/secret-santa/issues/43) and then second step would be inviting the attendees?

**Comment by mdwiltfong** [comment]
> Hi! I'm in the process of creating the wizard and just wanted to make sure that it is a two-step form. So the first step would have all the form inputs listed [here](https://github.com/mdwiltfong/secret-santa/issues/43) and then second step would be inviting the attendees? @mdwiltfong

That's right! On top, the second step needs to handle failed attempts to create sessions

**Comment by ivavay** [comment]
Okay!
